5_combinatorics/domino.cpp

```diff
@@ -0,0 +1,34 @@
+// https://acmp.ru/index.asp?main=task&id_task=328
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <cmath>
+#include <algorithm>
+
+using namespace std;
+
+int main() {
+    ifstream fin("input.txt");
+    ofstream fout("output.txt");
+
+    int n;
```

Comment:
It may help to pass all tests
```suggestion
    long long n;
```